### PR TITLE
Widget init size width wrong

### PR DESCRIPTION
**Pull request by Nedlinin**
This needs some testing and double checking.  Might be an opinion matter.  I'm not sure if a decision was made early on as to this functionality.. but, I don't see anything it breaks to remove the capping to the screen size.

Thoughts?

### CODE REVIEW

File: src/jquery.gridster.js [hunk 1]
```diff
@@ -3805,7 +3803,7 @@
 		for (var x = 1; x <= opts.cols; x++) {
 			var colWidth = ((x * opts.widget_base_dimensions[0]) + ((x - 1) * opts.widget_margins[0]));
 			styles += (opts.namespace + ' [data-sizex="' + x + '"] { width:' +
-			(full_width ? (this.$wrapper.width() - this.options.widget_margins[0] * 2) : colWidth > this.$wrapper.width() ? this.$wrapper.width() : colWidth ) + 'px; }\n');
+			(full_width ? (this.$wrapper.width() - this.options.widget_margins[0] * 2) : colWidth ) + 'px; }\n');
```
Comment (dsmorse) [inline src/jquery.gridster.js]: This change would cause errors when the grid is not the full width of the screen..

Comment (Nedlinin) [inline src/jquery.gridster.js]: Oh good call.  Never run gridster without it being full width so it didn't even cross my mind that was why it existed..

Do you have a suggested method to fixing the issue in the video I included in my issue? (#32) - Perhaps simply an option for width/height

File: dist/jquery.gridster.js [hunk 2]
```diff
@@ -4587,7 +4589,7 @@
 	 */
 	fn.get_responsive_col_width = function () {
 		var cols = this.cols || this.options.max_cols;
-		return (this.$el[0].scrollWidth - ((cols + 1) * this.options.widget_margins[0])) / cols;
+		return (this.$el[0].clientWidth - 3 - ((cols + 1) * this.options.widget_margins[0])) / cols;
```
Comment (cisso) [inline dist/jquery.gridster.js]: Why -3?

Comment (Nedlinin) [inline dist/jquery.gridster.js]: Honestly?  This is a really bad case of using a "magic number" and I have completely forgotten.  This PR had a couple of small issues that got it working quickly for a project but caused some other issues and I never got around to trying to fix the other issues it caused (we moved away from Gridster).

**Pull request closed without merging**